src/statement.js

```diff
@@ -115,11 +115,12 @@ class Statement {
     if (!fields) {
       return this;
     }
-    var fields = Array.isArray(fields) && arguments.length === 1 ? fields : Array.prototype.slice.call(arguments);
-    var map = this._order(fields);
-    map.forEach(function(dir, column) {
+    const _flds = Array.isArray(fields) && arguments.length === 1 ? fields : Array.prototype.slice.call(arguments);
```

Comment:
IIRC, rest arguments is more performant than `[].slice.call(arguments)`

Reply:
I'm pretty sure this is allowing for a string or array of strings.
I'm not sure why it only clones if there is more than one...

If I make it just rest args, it would be `[STRING]` or `[ [STRING, STRING] ]`. is that better?

Reply:
I'm also not really sure that messing with arguments is needed at all here. The docs all say an object or array of objects. it doesn't take variadic arguments.

So these could all be

```javascript
if (!fields) return this
Array.isArray(fields) ? fields : [fields];
```

Reply:
yea, I like that